modules/openapi-generator/src/main/resources/typescript-inversify/api.service.mustache

```diff
@@ -60,7 +60,7 @@ export class {{classname}} {
     public {{nickname}}({{#allParams}}{{paramName}}{{^required}}?{{/required}}: {{{dataType}}}, {{/allParams}}observe: any = 'body', headers: Headers = {}): {{#usePromise}}Promise{{/usePromise}}{{^usePromise}}Observable{{/usePromise}}<any> {
 {{#allParams}}
 {{#required}}
-        if (!{{paramName}}){
+        if ({{paramName}} == null){
```

Comment:
```suggestion
        if ({{paramName}} === null || {{paramName}} === undefined){
```

Reply:
Is there a specific reason for this change?
The semantics for both versions are identical. Cf. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#Loose_equality_using

Reply:
strict equality check is more explicit and easier to read, as it indicates that both cases are considered. I agree this does not change the functionality, but only the readability

Reply:
Ok, sure, that makes sense.